pkg/lib/files/files.go

```diff
@@ -347,6 +348,23 @@ func IgnoreSpecificFiles(absPaths ...string) IgnoreFn {
 	}
 }
 
+func GitIgnoreFn(gitIgnorePath string) (IgnoreFn, error) {
```

Comment:
I think the .cortexignore file should be excluded from the zip because I don't think the APIs should be redeployed if the .cortexignore file changes without any of the project files changing.

Reply:
`.cortexignore` will be excluded from the zip since we also use `files.IgnoreHiddenFiles` (so all files starting with . will be ignored)